config/rewrites.d/developer.rackspace.com.json

```diff
@@ -68,9 +68,15 @@
             "rewrite": false,
             "status": 301
         },
+         {
+            "from": "^\\/docs\\/cloud-monitoring\\/?(.*)$",
+            "to": "/how-to/rackspace-monitoring/$1",
+            "rewrite": false,
+            "status": 301
+        },
         {
-            "from": "^\\/docs\\/cloud-monitoring\\/v1\\/developer-guide\\/?(.*)$",
-            "to": "/docs/rackspace-monitoring/v1/developer-guide/$1",
+            "from": "^\\/docs\\/cloud-monitoring\\/getting-started\\/?(.*)$",
+            "to": "/docs/rackspace-monitoring/getting-started/$1",
```

Comment:
Also, this rule will never fire: the first one will redirect everything beneath `/docs/cloud-monitoring/`, including both `getting-started/` and `/v1/developer-guide/`.